code/game/objects/structures/crates_lockers/crates.dm

```diff
@@ -23,9 +23,16 @@
 	drag_slowdown = 0
 	var/obj/item/paper/fluff/jobs/cargo/manifest/manifest
 	var/radius_2 = 1.35
+	var/open_angle = 138 //azimuth angle for over 90 degree
+	var/static/list/animation_math //List with lists inside that contain all values needed for an animation
+	var/static/list/animation_index //associative list to get the index for the animation_math list that gets saved to anindex
+	var/anindex //index from animation_index for animation_math
+	var/crateexeption = FALSE //Set to true if the crate does not use the normal animation code and does not need a list
```

Comment:
```suggestion
	var/static/list/animation_math //assoc list of lists which contain all values needed for an animation
```

Reply:
reason for why i even went with a second list was cause i could not find a way to target  a list inside a list for things like list.len() or list.find()

Reply:
Still seems like i need to have a second list to get the index the direct association with the unique value does not seem to work